[2024-06-23T03:02:20.798+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-23T03:02:20.859+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_dag_with_encryption.extract_and_encrypt_data scheduled__2024-06-22T00:00:00+00:00 [queued]>
[2024-06-23T03:02:20.869+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_dag_with_encryption.extract_and_encrypt_data scheduled__2024-06-22T00:00:00+00:00 [queued]>
[2024-06-23T03:02:20.870+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 4
[2024-06-23T03:02:20.887+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): extract_and_encrypt_data> on 2024-06-22 00:00:00+00:00
[2024-06-23T03:02:20.907+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=4925) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-23T03:02:20.915+0000] {standard_task_runner.py:63} INFO - Started process 4928 to run task
[2024-06-23T03:02:20.912+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline_dag_with_encryption', 'extract_and_encrypt_data', 'scheduled__2024-06-22T00:00:00+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/bigquery.py', '--cfg-path', '/tmp/tmpx66bc6_5']
[2024-06-23T03:02:20.921+0000] {standard_task_runner.py:91} INFO - Job 83: Subtask extract_and_encrypt_data
[2024-06-23T03:02:21.086+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_pipeline_dag_with_encryption.extract_and_encrypt_data scheduled__2024-06-22T00:00:00+00:00 [running]> on host 64917a389d59
[2024-06-23T03:02:21.277+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_pipeline_dag_with_encryption' AIRFLOW_CTX_TASK_ID='extract_and_encrypt_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-22T00:00:00+00:00'
[2024-06-23T03:02:21.279+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-23T03:02:21.484+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-23T03:02:21.485+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-23T03:02:21.503+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_pipeline_dag_with_encryption, task_id=extract_and_encrypt_data, run_id=scheduled__2024-06-22T00:00:00+00:00, execution_date=20240622T000000, start_date=20240623T030220, end_date=20240623T030221
[2024-06-23T03:02:21.534+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-23T03:02:21.574+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-23T03:02:21.577+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
